Src/Base/AMReX_CoordSys.cpp

```diff
@@ -164,6 +164,7 @@ CoordSys::UpperIndex(const Real* point) const noexcept
     IntVect ix;
     for (int k = 0; k < AMREX_SPACEDIM; k++)
     {
+        // +1 for UpperIndex?
```

Comment:
btw, I was wondering if `UpperIndex` should have a +1

Reply:
it looks like it should, but it also appears that this isn't used anywhere